x-pack/legacy/plugins/maps/common/descriptor_types.d.ts

```diff
@@ -119,3 +118,36 @@ export type VectorLayerDescriptor = LayerDescriptor & {
   joins?: JoinDescriptor[];
   style?: unknown;
 };
+
+export type RangeFieldMeta = {
+  min: number;
+  max: number;
+  delta: number;
+  isMinOutsideStdRange?: boolean;
+  isMaxOutsideStdRange?: boolean;
+};
+
+export type CategoryMeta = {
```

Comment:
`CategoryMeta` and `CategoryFieldMeta` sound to similar and its not clear which is which. How about `Category` and `CategoryFieldMeta`?